CONTRIBUTING.md

```diff
@@ -292,10 +298,12 @@ checklist).
 2. Unit tests included and/or updated with changes?
 3. Command line build passes?
 4. Changes have been smoke-tested?
+5. Testing instructions included?
 
 ### Reviewer Checklist
 
 1. Changes appear to address issue?
 2. Appropriate unit tests included?
 3. Code style and in-line documentation are appropriate?
 4. Commit messages meet standards?
+5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue)
```

Comment:
Suggest rewording this: "If this PR closes an associated issue, has that issue been labeled `unverified`?

Reply:
Hehe, I actually had it worded that way originally. I decided it was too verbose if you're scanning over it every time you file a PR though. I also figured that the instructions about labelling the issue as unverified were more important than the qualifier?

Reply:
Could shorten to:
Associated issued labelled `unverified`? (if PR closes issue)

Reply:
I'm OK with what Andrew has.